.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+<!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
+
+**Is this a request for help** (If yes, you should instead post it in StackOverflow, Slack channel, or mailing list, see http://kubernetes.io/docs/troubleshooting/.):
```

Comment:
Perhaps something like: **Is this a request for help** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.)